app/views/job_invocations/_form.html.erb

```diff
@@ -113,7 +113,7 @@
     </div>
 
     <div class="form-group">
-      <%= add_label({ :label => _('Type of query'), :label_help => _("Type has impact on when is the query evaluated to hosts.<br><ul><li><b>Static</b> - evaluates just after you submit this form</li><li><b>Dynamic</b> - evaluates just before the execution is started, so if it's planed in future, targeted hosts set may change before it</li></ul>") }, f, :targetting_type) %>
+      <%= add_label({ :label => _('Type of query'), :label_help => _("Type has impact on when is the query evaluated to hosts.<br><ul><li><b>Static</b> - evaluates just after you submit this form</li><li><b>Dynamic</b> - evaluates just before the execution is started, so if it's planed in future, targeted hosts set may change before it</li></ul>") }, f, :targeting_type) %>
```

Comment:
Doesn't there actually have to be an element with the same id (`targeting_type`) in order for this change to actually do anything? (https://www.w3schools.com/tags/tag_label.asp)

Reply:
Ah you're right, for some reason I mis-read the radio buttons and thought they had the same `id` attr as the form label.

![image](https://user-images.githubusercontent.com/32345766/94852822-efc92e00-0486-11eb-8777-eb93a324ddf2.png)

Will close this off